### DISCUSSION TITLE
Request for Time Acceleration Mode

### DISCUSSION THREAD

**Discussion by Abendlied** [Try/OpenGothic#658] [General]
For testing purposes, it would be beneficial in observation situation to accelerate game time. Can we have a mode (e.g. marvin mode) to accelerate time? With parameter would be awesome, so we could change between slightly faster and hyper acceleration like 1 day in 5 sec. or so.

`set time %d %d` only changes time, does not do what I want it to do.
`ztimer multiplier X` would help.

**Comment by Try** [comment]
Hi, @Abendlied !

For marvin command that already exist in original game, you can put request here: https://github.com/Try/OpenGothic/issues/215

Done in 02538d8ffb297b502b0687f35e32ba15aa7e46fb

**Comment by Abendlied** [comment]
Thank you @Try, works perfectly!
